### DISCUSSION TITLE
Is there a way to adjust the width of the tab page to make it narrower?

### DISCUSSION THREAD

**Discussion by xMonody** [kovidgoyal/kitty#7482] [General]
Is there a way to adjust the width of the tab page to make it narrower?
![88d849d356a2a1b83504cc73301c1ba4](https://github.com/kovidgoyal/kitty/assets/76483540/86242629-c7ec-401b-b360-5f3ee24f847f)

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.tab_title_template

**Comment by xMonody** [comment]
I see, thank you.
